Neos.ContentRepository.Core/Classes/ContentRepository.php

```diff
@@ -127,7 +125,15 @@ public function handle(CommandInterface $command): CommandResult
             $eventsToPublish->expectedVersion,
         );
 
-        return $this->eventPersister->publishEvents($eventsToPublish);
+        $this->eventPersister->publishEvents($this, $eventsToPublish);
+        return new class {
+            /**
+             * @deprecated backwards compatibility layer
+             */
+            public function block(): void
+            {
+            }
+        };
```

Comment:
if we were to merge this like this - which i think is not necessary - we should use an actual class so its inspectable via phpstorm :) But i dont think we need b/c layer for new code